### ISSUE TITLE
[Feature] Send console output to terminal

### ISSUE THREAD

**Issue by IanVS** [storybookjs/test-runner#155]
### Describe the feature

I've hit a situation where tests are passing in the browser, but not in the test-runner.  Without having some way to inspect the state of my story in the test-runner, it's very difficult to troubleshoot what is happening.  I propose one of two things:

1) Send all console output to the terminal in the case of failing tests.  There could be useful information which can help debug the reason for the failure, so it should be provided to the user.  Passing tests should not output console warnings or logs.

2) Add a flag to send all console output to the terminal, for passing and failing tests.  I think this is less desirable, because it's another flag that people have to learn.

**Comment by IanVS** [comment]
I tried poking around to find a way to make this happen, without luck so far.  But as a workaround, I can add a `.storybook/test-runner.cjs` with the contents:

```js
module.exports = {
  async preRender(page) {
    page.on('console', (err) => {
      throw new Error(err.text());
    });
  },
};
```

and I get console output, at least as a debugging tool.

**Comment by yannbf** [comment]
Closed via #157 ! Currently available in 0.6.0-next.1 :D
